docs/how-to-use-customized-configuration.md

```diff
@@ -0,0 +1,245 @@
+   * Example
+     ```console
+     kubectl create secret generic scalardb-credentials-secret \
+       --from-literal=SCALAR_DB_USERNAME=postgres \
+       --from-literal=SCALAR_DB_PASSWORD=postgres
+     ```
+
+1. Set the `Secret` name to the following keys in the custom values file.  
+   * Keys
+     * `scalardb.secretName` (Scalar DB)
+     * `ledger.secretName` (Scalar DL Ledger)
+     * `auditor.secretName` (Scalar DL Auditor)
+   * Example
+     * Scalar DB
+       ```yaml
+       scalardb:
+         secretName: "scalardb-credentials-secret"
+       ```
+     * Scalar DL Ledger
+       ```yaml
+       ledger:
+         secretName: "ledger-credentials-secret"
+       ```
+     * Scalar DL Auditor
+       ```yaml
+       auditor:
+         secretName: "auditor-credentials-secret"
+       ```
+
+1. Deploy Scalar products with the above custom values file.  
+   After deploying Scalar products, the Go template strings (environment variables) are replaced by the values of the `Secret`.
+   * Example
+       * Custom values file
+         ```yaml
+         scalardb:
+           databaseProperties: |
+             scalar.db.contact_points=jdbc:postgresql://postgresql-scalardb.default.svc.cluster.local:5432/postgres
+             scalar.db.username={{ default .Env.SCALAR_DB_USERNAME "" }}
+             scalar.db.password={{ default .Env.SCALAR_DB_PASSWORD "" }}
+             scalar.db.storage=jdbc
+         ```
+       * Properties file in containers
+         ```properties
+         scalar.db.contact_points=jdbc:postgresql://postgresql-scalardb.default.svc.cluster.local:5432/postgres
+         scalar.db.username=postgres
+         scalar.db.password=postgres
+         scalar.db.storage=jdbc
+         ```
+
+   Please refer to the [Getting Started with Scalar Helm Charts](./getting-started-scalar-helm-charts.md) for more details on how to use each Helm Chart.
+
+## Mount arbitrary files like key and certificate files to the container in Scalar DL Helm Charts
+
+You can mount any files to the container when you use the Scalar DL Helm Charts. For example, you need to mount the key and certificate files to run the Scalar DL Auditor.
+
+* Configuration example
+    * Scalar DL Ledger
+      ```yaml
+      ledger:
+        ledgerProperties: |
+          ...
+          scalar.dl.ledger.proof.enabled=true
+          scalar.dl.ledger.auditor.enabled=true
+          scalar.dl.ledger.proof.private_key_path=/keys/private-key
+      ```
+    * Scalar DL Auditor
+      ```yaml
+      auditor:
+        auditorProperties: |
+          ...
+          scalar.dl.auditor.cert_path=/keys/certificate
+          scalar.dl.auditor.private_key_path=/keys/private-key
+      ```
+
+In this example, you need to mount a **private-key** and a **certificate** file under the `/key` directory in the container. And, you need to mount files named `private-key` and `certificate`. You can use `extraVolumes` and `extraVolumeMounts` to mount these files.
```

Comment:
Probably the `/keys` is a correct one?

```suggestion
In this example, you need to mount a **private-key** and a **certificate** file under the `/keys` directory in the container. And, you need to mount files named `private-key` and `certificate`. You can use `extraVolumes` and `extraVolumeMounts` to mount these files.
```

Reply:
You are right. It's typo...
Thank you for catching it!